### PR TITLE
fix getPruneCounts query string params

### PR DESCRIPTION
**Pull request by lts20050703**

### CODE REVIEW

File: helpers/guilds/getPruneCount.ts [hunk 1]
```diff
@@ -3,23 +3,16 @@ import type { Bot } from "../../bot.ts";
...
-
-  if (options) {
-    url += "?";
-
-    if (options.days) url += `days=${options.days}`;
-    if (options.includeRoles) url += `&include_roles=${options.includeRoles}`;
-  }
+  const queries = [];
+  if (options?.days) queries.push(`days=${options.days}`);
+  if (options?.includeRoles) queries.push(`include_roles=${options.includeRoles}`);
```
Comment (itohatweb) [inline helpers/guilds/getPruneCount.ts:10]:
```suggestion
  let url = bot.constants.endpoints.GUILD_PRUNE(guildId);

  if (options) {
    url += "?";

    if (options.days) url += `days=${options.days}`;
    if (options.includeRoles) url += `&include_roles=${options.includeRoles}`;
  }
```

File: helpers/guilds/getPruneCount.ts [hunk 2]
```diff
@@ -3,23 +3,16 @@ import type { Bot } from "../../bot.ts";
...
 
   const result = await bot.rest.runMethod(
     bot.rest,
     "get",
-    url,
+    `${bot.constants.endpoints.GUILD_PRUNE(guildId)}${queries.length ? `?${queries.join("&")}` : ""}`,
```
Comment (itohatweb) [inline helpers/guilds/getPruneCount.ts:15]:
```suggestion
    url,
```

**Review by itohatweb — CHANGES_REQUESTED** [review]
Those changes have a bad performance impact

File: helpers/guilds/getPruneCount.ts (hunk 1 above)
Comment (lts20050703) [reply helpers/guilds/getPruneCount.ts:10]: @itohatweb
Is url? valid? (everything is optional)
Is url?&after valid? (everything is optional)

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**